R/standardise_titles.R

```diff
@@ -11,13 +11,13 @@
 #' It trims white spaces from the start, middle and end of the strings.
 #' Removes ambiguous punctions and symbols from strings.
 #' All the strings are transformed into to ASCII character encoding.
-#' Written numbers in ordinal form are transformed into numerical form.  
+#' Written numbers in ordinal form are transformed into numerical form.
 #' If a google API key is provided as an argument, the function detects
 #' strings in other languages and translates them to English.
 #' @return A capitalised, trimmed and standardised string
 #' @importFrom textclean add_comma_space mgsub
 #' @importFrom english ordinal words
-#' @importFrom stringr str_count
+#' @importFrom stringr str_count str_squish str_to_title
```

Comment:
Looks like the dependency here is stringr not stringi?

Reply:
Both packages are used for the function. The function `stri_trans_general` from stringi is used to transform strings to ASCII characters